sdk/tests/deqp/functional/gles3/es3fSyncTests.js

```diff
@@ -33,6 +33,7 @@ goog.scope(function() {
     var deString = framework.delibs.debase.deString;
 
     /** @const {number} */ es3fSyncTests.NUM_CASE_ITERATIONS = 5;
+    /** @const {number} */ es3fSyncTests.MAX_NUM_VERIRY = 10;
```

Comment:
Will this introduce flakiness into the test?  because there is really no guarantee that it returns in 10 frames.